scripts/create_polygons.py

```diff
@@ -4,20 +4,21 @@
 from collections import Counter
 from urllib.parse import urlparse
 
-import aws_helper as aws_helper
+import aws_helper
 from linz_logger import get_log
-from osgeo import gdal
+from osgeo import gdal  # pylint: disable=import-error
```

Comment:
Maybe we should add a comment here saying that this library is not imported with `Poetry` because of import issues, but included in the Docker image we use.